e2e/sharedFlows/switchPowerMode.js

```diff
@@ -10,10 +10,10 @@ export default async function switchPowerMode() {
   const settingsDrawerMenuItem = element( by.id( "settings" ) );
   await waitFor( settingsDrawerMenuItem ).toBeVisible().withTimeout( 10000 );
   await settingsDrawerMenuItem.tap();
-  // Tap the settings radio button for advanced interface mode
```

Comment:
Have you tested this? Pretty sure this will break e2e. Because a press on an already set advanced mode setting as is now changes nothing, bit a press on a radio does toggle every time.